hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/contracts/operation/HederaSelfDestructOperationV046.java

```diff
@@ -74,6 +74,9 @@ public OperationResult execute(final MessageFrame frame, final EVM evm) {
         final var updater = (HederaStackedWorldStateUpdater) frame.getWorldUpdater();
         final var beneficiaryAddress = Words.toAddress(frame.getStackItem(0));
         final var toBeDeleted = frame.getRecipientAddress();
+        if (frame.isStatic()) {
```

Comment:
This check is already done during `super.execute()`. Why does it need to be added here?